### ISSUE TITLE
None of the base16 themes deal well with bufferline. Selected buffer not visible

### ISSUE THREAD

**Issue by paletochen** [base16-project/base16-helix#4]
Hi,
I noticed that none of these base16 themes highlight the buffer selected in the bufferline, making them not suitable when working in helix and multiple buffers at once.

Screenshot where you can see the bufferline at the top and no way how to distinguish the active one

![image](https://user-images.githubusercontent.com/109062850/218248964-bd926436-3c6d-4154-8cfc-6273893c536f.png)

Below any other theme which shows it

![image](https://user-images.githubusercontent.com/109062850/218249071-ce83068a-3f89-4bc6-89df-954bb5aa032a.png)

Could you please have a look at it?

Thanks

**Comment by paletochen** [comment]
One recommendation

```toml
"ui.bufferline" = { fg = "base04", bg = "base00" }
"ui.bufferline.active" = { fg = "base00", bg = "base03", modifiers = ["bold"]}
```

This would look like in base16-google-dark (as an example) like below:

![image](https://user-images.githubusercontent.com/109062850/218251943-c3ce7ae7-388f-42e7-aa1c-4c5229d63f6e.png)

**Comment by krgn** [comment]
Hi @paletochen! Thank you for the suggestions. Could you pull [this PR](https://github.com/tinted-theming/base16-helix/pull/5) and see if the rendered themes fit your needs? Thanks

**Comment by paletochen** [comment]
> Hi @paletochen! Thank you for the suggestions. Could you pull [this PR](https://github.com/tinted-theming/base16-helix/pull/5) and see if the rendered themes fit your needs? Thanks

Hi @krgn 
I looks perfect! :+1:

**Comment by krgn** [comment]
Fixed in #5
